index.js

```diff
@@ -141,5 +91,29 @@ module.exports = {
         }));
 
         return this.mergeTrees(trees, { overwrite: true });
+    },
+
+    _transformLocale: function (locale, result) {
+        var data = ['/*jslint eqeq: true*/', 'export default {'];
+
+        data.push('  locale: "' + locale + '",');
+
+        if (result.parentLocale) {
+            data.push('  parentLocale: "' + result.parentLocale + '",');
+        }
+
+        if (result.fields) {
+            data.push('  fields: ' + serialize(result.fields) + ',');
```

Comment:
You can build up a JSON/JavaScript object and then call `serialize()` once for the whole thing.